src/index.js

```diff
@@ -53,7 +53,9 @@ function waitForId(fn) {
   }
 }
 
-// We define everything relevant to our bot inside this function.
+/* 
+ * We define everything relevant to our bot inside this function.
```

Comment:
This is like not true anymore right? Maybe something like "sets up the bot and calls the botLoop function every time an animation frame is drawn"